http/src/main/java/com/github/awsjavakit/http/updatestrategies/TokenCacheUpdateStrategy.java

```diff
@@ -3,7 +3,27 @@
 import com.github.awsjavakit.http.token.OAuthTokenEntry;
 import java.util.function.Supplier;
 
-public interface TokenCacheUpdateStrategy<T> {
+/**
+ *  Given a function that provides a new Token and updates some cache (updateCache),
+ *  and a function that provides an already cached entry,
+ *  then this function decides how will these functions will be combined to provide a valid token.
```

Comment:
"how these functions will be combined" not "how will these functions will be combined" :)